### ISSUE TITLE
make e2e test work

### ISSUE THREAD

**Issue by ludydoo** [commonpool/commonpool#1]
e2e tests are broken right now. 
find a way to bypass the oidc auth with protractor

**Comment by nilueps** [comment]
A bit lost tbh XD

**Comment by nilueps** [comment]
` Failed: Error while waiting for Protractor to sync with the page: "both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor's bootstrapping.  See http://git.io/v4gXM for details"`

any lead on how to wrangle this?

**Comment by ludydoo** [comment]
yes. The problem is that protractor is being redirected to keycloak for authentication, but doesn't know how to handle this.
